.travis.yml

```diff
@@ -0,0 +1,10 @@
+sudo: required
+services:
+  - docker
+script:
+  - docker login -u $DOCKER_USER -p $DOCKER_PASS
```

Comment:
It looks like you forgot to add DOCKER_USER and DOCKER_PASS to this config.
To do that you have to install Travis app(https://github.com/travis-ci/travis.rb)
And run 
`travis encrypt DOCKER_USER=yourUser --add`
`travis encrypt DOCKER_PASS=yourPassw0rd --add`